tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,300 @@
+
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Feature                 | Compatibility            | Mobile                   | Forward+                 |
++=========================+==========================+==========================+==========================+
+| Sub-surface scattering  | ❌ Not supported.        | ❌ Not supported.        | ✔️ Supported.            |
+|                         |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+
+Shader features
+~~~~~~~~~~~~~~~
+
+See :ref:`doc_shading_reference` for more information.
+
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Feature                 | Compatibility            | Mobile                   | Forward+                 |
++=========================+==========================+==========================+==========================+
+| Per-instance shader     | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
+| uniforms                |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Screen texture          | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Depth buffer            | ✔️ Supported.            | ✔️ Supported.            | ✔️ Supported.            |
+|                         |                          |                          |                          |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Normal/Roughness buffer | ❌ Not supported.        | ✔️ Supported.            | ✔️ Supported.            |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Unlit color buffer      | ❌ Not supported.*       | ❌ Not supported.*       | ❌ Not supported.*       |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Lighting buffer         | ❌ Not supported.*       | ❌ Not supported.*       | ❌ Not supported.*       |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Stencil buffer          | ❌ Not supported.        | ⚠️ Planned.              | ⚠️ Planned.              |
++-------------------------+--------------------------+--------------------------+--------------------------+
+| Custom buffer           | ❌ Not supported.        | ❌ Not supported.        | ❌ Not supported.        |
```

Comment:
Documenting the features that are *not* supported at all could be out of scope for this page.

Reply:
Per-instance shader uniforms are supported in Compat as of a few days ago in 4.4.

Features that are available in all or in none of the backends IMO should be out of scope here, because otherwise the page just ends up a giant table of all hypothetical features a renderer could have.

Reply:
I would make an exception for *sets* of features where one one of the set is supported by all renderers. For example, I want to make clear that the normals/rough texture is not supported on Compatibility, but the depth texture is. I also want to list any antialiasing methods that are supported on all renderers, because most are not.

Either way the completely unsupported ones in this table are clearly out of scope.

Reply:
Ya, your examples make sense. But the ones highlighted by this comment should be removed. 

Also, as a side note, stencil should be supported in all three renderers eventually. And custom buffer will likely be supported even before stencil! But I wouldn't mention them here until they are implemented.